modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1674,6 +1673,27 @@ private void _deleteFromTable(
 		FinderCacheUtil.clearDSLQueryCache(dbTableName);
 	}
 
+	private void _deleteLocalizationValues(
+		ObjectDefinition objectDefinition, long objectEntryId) {
+
+		DynamicObjectDefinitionLocalizationTable
+			dynamicObjectDefinitionLocalizationTable =
+				DynamicObjectDefinitionLocalizationTableFactory.create(
+					objectDefinition, _objectFieldLocalService);
+
+		if (dynamicObjectDefinitionLocalizationTable == null) {
+			return;
+		}
+
+		runSQL(
+			StringBundler.concat(
+				"delete from ",
+				dynamicObjectDefinitionLocalizationTable.getName(), " where ",
+				dynamicObjectDefinitionLocalizationTable.
+					getForeignKeyColumnName(),
+				" = ", objectEntryId));
+	}
+
```

Comment:
I saw that the method "_deleteFromTable()" has this code after running the SQL: `FinderCacheUtil.clearDSLQueryCache(dbTableName);`

I don't know if we should add it, just sharing with you.